op-devstack/presets/multi_with_synctester.go

```diff
@@ -0,0 +1,85 @@
+package presets
+
+import (
+    "github.com/ethereum-optimism/optimism/op-devstack/devtest"
+    "github.com/ethereum-optimism/optimism/op-devstack/dsl"
+    "github.com/ethereum-optimism/optimism/op-devstack/shim"
+    "github.com/ethereum-optimism/optimism/op-devstack/stack"
+    "github.com/ethereum-optimism/optimism/op-devstack/stack/match"
+    "github.com/ethereum-optimism/optimism/op-devstack/sysgo"
+)
+
+// MultiWithSyncTester provisions two L2 CL nodes, each backed by a distinct SyncTester EL endpoint,
+// and exposes handles for tests.
+type MultiWithSyncTester struct {
+	Minimal
+
+	SyncTester *dsl.SyncTester
+
+	// Two independent SyncTester-backed EL endpoints
+	SyncTesterL2ELA *dsl.L2ELNode
+	SyncTesterL2ELB *dsl.L2ELNode
+
+	// Two CLs, each wired to a different SyncTester EL endpoint
+	L2CL_A *dsl.L2CLNode
+	L2CL_B *dsl.L2CLNode
+}
+
+// WithMultiWithSyncTester composes a minimal system with a SyncTester service, then adds two
+// SyncTester EL endpoints and two CL verifier nodes connected to them.
+func WithMultiWithSyncTester() stack.CommonOption {
+	// Build a combined option sequence
+	opt := stack.Combine[*sysgo.Orchestrator]()
```

Comment:
These initialization process should stay at `op-devstack/sysgo`. Please refer to `DefaultSimpleSystemWithSyncTester`. At `op-devstack/preset`(here), only the DSL wrapper `NewMultiWithSyncTester` should exist.

We may need something like `DefaultSimpleSystemWithMultiSyncTesterIDs`.